app/views/publisher/start.html

```diff
@@ -14,7 +14,8 @@
         <div class="govuk-grid-column-full">
             <h1 class="govuk-heading-xl">Your forms</h1>
             {{ govukButton({
-                text: "Create a new form"
+                text: "Create a new form",
+                href: '../form-designer/form-index'
```

Comment:
Think this should point to either `../form-designer/form-create-a-form.html` or `../form-designer/form-home.html` (this was one of my changes I think so my fault)